src/test/java/org/example/MyTests.java

```diff
@@ -0,0 +1,187 @@
+import org.testng.annotations.Test;
+
+public class MyTests extends BaseTests {
+
+    @Test
+    public void loginTest() throws InterruptedException {
+        driver.findElement(By.className("wu_entr")).click();
+        Thread.sleep(1000);
+        driver.findElement(By.xpath(".//div[contains(@class, 'signin-with-ek')]")).click();
+        Thread.sleep(1000);
+        driver.findElement(By.className("ek-form-control")).click();
+        Thread.sleep(1000);
+        WebElement emailField = driver.findElement(By.xpath(".//input[@name='l_']"));
+        emailField.sendKeys("oleg123@mailinator.com");
+        WebElement passwordField = driver.findElement(By.xpath(".//input[@name='pw_']"));
+        passwordField.clear();
+        passwordField.sendKeys("1234qwer", Keys.ENTER);
+        Thread.sleep(2000);
+
+        Assertions.assertThat(driver.findElement(By.id("mui_user_login_row")).isDisplayed())
+                .as("User icon is shown on the right corner")
+                .isTrue();
+    }
+
+    @Test
+    public void loginWithIncorrectPassword() throws InterruptedException {
+        driver.findElement(By.className("wu_entr")).click();
+        Thread.sleep(1000);
+        driver.findElement(By.xpath(".//div[contains(@class, 'signin-with-ek')]")).click();
+        Thread.sleep(1000);
+        driver.findElement(By.className("ek-form-control")).click();
+        Thread.sleep(1000);
+        WebElement emailField = driver.findElement(By.xpath(".//input[@name='l_']"));
+        emailField.sendKeys("oleg123@mailinator.com");
+        WebElement passwordField = driver.findElement(By.xpath(".//input[@name='pw_']"));
+        passwordField.clear();
+        passwordField.sendKeys("12345qwer", Keys.ENTER);
+        Thread.sleep(2000);
+
+        Assertions.assertThat(driver.findElement(By.xpath(".//div[contains(@class, 'ek-form-text')]")).getText())
+                .as("Message about wrong password is shown")
+                .isEqualTo("Пароль указан неверно!");
+    }
+
+    @Test
+    public void addTextInProfileInfo() throws InterruptedException {
+        driver.findElement(By.className("wu_entr")).click();
+        Thread.sleep(1000);
+        driver.findElement(By.xpath(".//div[contains(@class, 'signin-with-ek')]")).click();
+        driver.findElement(By.className("ek-form-control")).click();
+        Thread.sleep(1000);
+        WebElement emailField = driver.findElement(By.xpath(".//input[@name='l_']"));
+        emailField.sendKeys("oleg123@mailinator.com");
+        WebElement passwordField = driver.findElement(By.xpath(".//input[@name='pw_']"));
+        passwordField.clear();
+        passwordField.sendKeys("1234qwer", Keys.ENTER);
+        Thread.sleep(1000);
+
+        driver.findElement(By.className("info-nick")).click();
+        driver.findElement(By.className("user-menu__edit")).click();
+        WebElement dedcriptionField = driver.findElement(By.name("p_[SelfDescription]"));
+        dedcriptionField.sendKeys("Hello");
```

Comment:
let's clear dedcriptionField before typing "Hello" in case if description is not empty